### ISSUE TITLE
internal types

### ISSUE THREAD

**Issue by freever** [Husqvik/GraphQlClientGenerator#49]
Hi @Husqvik 

This is a fantastic library, thanks so much. Graphql basically brings a type system to APIs and it feels awful having to use strings all the time to mediate between C# types and graph types. I am using your library to generate builders against my schema so I never have to type any strings, and for that the word is a better place. 

One query I have is about the access modifiers for generated classes. I have found that all the data classes ("#region data classes") conflict with the names of my entities and queries so I have written some ugly string parsing to convert them to internal. Now I can access only the clean graph builder stuff without any conflicts. 

I was wondering if there is a need for these classes to be public, or if they should perhaps have a more restricted access? 

Anyway thanks again.

**Comment by bddckr** [comment]
The console app has support to specify a namespace, at which point you should be able to easily prevent name clashes by using a specific namespace for the auto-generated output.
https://github.com/Husqvik/GraphQlClientGenerator/blob/772552c401ff1ddd97cbe01814d77b5bc97cff1c/src/GraphQlClientGenerator.Console/GraphQlCSharpFileHelper.cs#L31

**Comment by Husqvik** [comment]
It's reasonable request I'll add the option to the generator. Original idea was to have like a package generated for the client classes which normally should expose only public classes but I guess for more advanced use there can be numerous reasons to choose other models/approaches.

**Comment by freever** [comment]
@bddckr Thanks for the tip. I was using what was available in the nuget package, but that meant I needed to write my own crappy version of the FileHelper logic. I've switched to cloning the repo and running the console app from the command line. It does everything I need. 

@Husqvik I had the problem because I was running tests against my graphql API using queries generated by the builder. Some better organization of projects would have alleviated the issue.
